src/robusta/runner/config_loader.py

```diff
@@ -231,6 +231,10 @@ def __reload_playbook_packages(self, change_name):
                 self.registry.set_playbooks(playbooks_registry)
                 self.registry.set_sinks(sinks_registry)
 
+                telemetry = self.registry.get_telemetry()
+                telemetry.sinks_findings_count = { k:0 for (k, _) in sinks_registry.get_all().items() }
```

Comment:
We can also use a defaultdict here:
`telemetry.sinks_findings_count = defaultdict(lambda: 0)`

But it will have only sinks which have at least one message